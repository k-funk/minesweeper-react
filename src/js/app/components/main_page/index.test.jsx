import React from 'react';
import { shallow } from 'enzyme';
import shallowToJson from 'enzyme-to-json';

import MainPage from './index';


describe('outputs the expected tree when', () => {
  let wrapper;

  test('(default', () => {
    wrapper = shallow((
      <MainPage />
    ));
  });

  afterEach(() => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
