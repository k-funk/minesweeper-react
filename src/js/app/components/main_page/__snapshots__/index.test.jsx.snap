// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`outputs the expected tree when (default 1`] = `
<div
  className=""
>
  <div>
    notStarted
  </div>
  <div>
    Mines Left/flags left to place: 0 FIXME
  </div>
  <Board
    height={4}
    mines={2}
    width={4}
  />
</div>
`;
